### PR TITLE
[LIGO-549] Change style

### PR DESCRIPTION
**Pull request by sashasashasasha151**
Problem: EthereumStudio has its own style but we
need to use ligo style

Solution: change colors

[//]: # (This is a template of a pull request template.)
[//]: # (You should modify it considering specifics of a particular repository and put it there.)
[//]: # (Comments like this are meta-comments, they shouldn't be present in the final template.)
[//]: # (Comments starting with '

[//]: # (Here you can add a link to the corresponding issue tracker, e. g. https://issues.serokell.io/issue/AD-)
[//]: # (For GitHub/GitLab issues it is better to use just hash symbol or exclamation mark as it is more resistant to repo movements)
[//]: # (In this case please also prefix the link with "Resolves" keyword)
[//]: # (See https://docs.gitlab.com/ee/user/project/issues/managing_issues.html#closing-issues-automatically)
[//]: # (See https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords)
#### Related changes (conditional)

- Tests
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

[//]: # (Add more docs here if you have them in the repository)
- Documentation
  - [ ] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)
    - Haddock

[//]: # (Mostly for public repositories)
[//]: # (Recording changes is optional, depends on repository, useful for some libs)
- Public contracts
  - [ ] Any modifications of public contracts comply with the [Evolution
  of Public Contracts](https://www.notion.so/serokell/Evolution-of-Public-Contracts-2a3bf7971abe4806a24f63c84e7076c5) policy.
  - [ ] I added an entry to the [changelog](../tree/master/CHANGES.md) if my changes are visible to the users
        and
  - [ ] provided a migration guide for breaking changes if possible

#### Stylistic guide (mandatory)

[//]: # (Update link to style guide if necesary or remove if it's not present)

- [ ] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).
- [ ] My code complies with the [style guide](../tree/master/docs/code-style.md).

### CODE REVIEW

File: ligo-webide-frontend/ligo-ide/src/scss/index.scss [hunk 1]
```diff
@@ -46,8 +46,9 @@ $jumbotron-padding: 1.5rem;
 $modal-md: 720px;
 $modal-content-bg: $background;
 $modal-header-border-color: $black;
-$modal-backdrop-opacity: .7;
-$btn-disabled-opacity: .3;
+$modal-backdrop-opacity: .6;
+$modal-backdrop-bg: #d0e3ff;
```
Comment (pgujjula) [inline ligo-webide-frontend/ligo-ide/src/scss/index.scss]: where did you get this hexcode (and the other hexcode literals)?

**Comment by pgujjula** [comment]
I notice there's a lot of hexcode literals, many of them repeated. It would be nice to have them as named constants, or have some more documentation about how they were generated.

Otherwise the changes look good on my machine, so that's my only comment.

File: ligo-webide-frontend/ligo-ide/src/scss/index.scss (hunk 1 above)
Comment (sashasashasasha151) [reply ligo-webide-frontend/ligo-ide/src/scss/index.scss]: Used ligo blue colour but more soft, so it was just random softer color based on ligo blue color

**Review by pgujjula — APPROVED** [review]
LGTM

**Pull request merged**